service-base-utils-bld-os/pom.xml

```diff
@@ -0,0 +1,40 @@
+<!--  ************************************** -->
+<!--  (C) Copyright IBM Corp. 2020, 2020     -->
+<!--  SPDX-License-Identifier: Apache-2.0    -->
+<!--  ************************************** -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.ibm.watson.health.services</groupId>
+		<artifactId>service-base-utils-parent-os</artifactId>
+		<version>${revision}</version>
+<!-- 		<version>1.0.0-SNAPSHOT</version> -->

```

Comment:
Probably worth just cleaning up this commented line.

Reply:
Agreed. I will clean this up. Good catch.